Emby.Server.Implementations/Activity/ActivityLogEntryPoint.cs

```diff
@@ -92,28 +93,28 @@ public void Run()
             _appHost.ApplicationUpdated += _appHost_ApplicationUpdated;
         }
 
-        void _deviceManager_CameraImageUploaded(object sender, GenericEventArgs<CameraImageUploadInfo> e)
+        async void _deviceManager_CameraImageUploaded(object sender, GenericEventArgs<CameraImageUploadInfo> e)
```

Comment:
Do these method names come from emby? I would rename them to `OnDeviceManagerCameraImageUpload` to respect language best practices.

Reply:
Yes there come from Emby

Reply:
IMO they should/could be mass renamed in one big PR.